openmoc/materialize.py

```diff
@@ -272,7 +273,9 @@ def load_openmc_mgxs_lib(mgxs_lib, geometry=None):
 
                 # If the user filled the Cell with a Material, clone it
                 if material != None:
-                    material = material.clone()
+                    # FIXME!!!
+#                    material = material.clone()
+                    material = material
```

Comment:
Yes, this is something I'm still working on, I'll post here once it is fixed. Material cloning is needed here when one has an OpenMC MGXS library tallied on cells when multiple OpenMC `Cells` use the same `Material` (*e.g.*, different spatial discretizations, such as rings an sectors in a fuel pin). In this case, we must create `Material` *clones* for each `Cell` in the coresponding OpenMOC `Geometry`. 

This works just fine if one simply runs a single eigenvalue / fixed source calculations in a script, such as is the case in all of our example inputs. However, I need to put in a little more sophisticated logic here to permit one to run multiple calculations in the same input (*i.e.*, a fixed source calculations for each SPH iteration).